rosidl_generator_c/test/test_interfaces.c

```diff
@@ -378,9 +378,6 @@ int test_string_arrays(void)
   bool res = false;
   rosidl_generator_c__msg__StringArrays * strings = rosidl_generator_c__msg__StringArrays__create();
 
-  res = rosidl_generator_c__msg__StringArrays__init(strings);
```

Comment:
We could also add `EXPECT_NE(strings, NULL);`

Reply:
Done!